test/account_test.py

```diff
@@ -0,0 +1,6 @@
+import unittest
+
+
+class AccountTest(unittest.TestCase):
+    def test_account_me(self):
+        pass
```

Comment:
No, this is not good. Either write the test now or don't create this harness. This leads to a false sense of safety when running tests, where one thinks they're passing, but in fact, there are no tests.

Reply:
@edpin  All of the pre-existent tests were just a pass, we just reorganized it now, as soon as we refactor the SDK, we will come back to fix the tests

Reply:
Understood, but it was a bad idea to have them blank/pass in the first place. Not sure who put it there, but we can fix it now. We need to remove them and add them one-by-one later. Better no tests than a false sense of safety.